src/components/quizOne.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../styles/root.css" />
+	</head>
+	<body>
+		<div
+			class="InstallationDomMainContainer TypoHtag card-container-bg flex-space-evenly"
+		>
+			<div
+				class="padding-small-card shade classA translateOne"
+			>
+			 <div
+				class="padding-small-card shade flex-column-al-center translateOne margin-normal-bottom back-btn"
+		>score: 2/5 </div>
+					<div
+						class="quiz-container padding-small-card shade"
+					>
+						<p class="padding-normal">
+							Lorem ipsum, dolor sit amet consectetur
+							adipisicing elit. Eius cumque maiores a alias
+							repellat porro odit fugit possimus obcaecati
+							quasi quaerat adipisci iusto dignissimos, iure
+							quo reiciendis atque recusandae! Eveniet.
+						</p>
+					</div>
+					<div class="padding-normal card-selector-text">
+						<div class="option-container">
+							<button class="options quiz-container padding-small-card imgTransition" value="true">Option 1</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 2</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 3</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 4</li>
```

Comment:
suggestion: camelcase should be used in js I think, it's not wrong to use here as it will get the work done, but Animesh once mentioned about it so I'm just sharing this.

Reply:
I'll change that also.